libutils/logging.c

```diff
@@ -173,7 +173,7 @@ static void LogToConsole(const char *msg, LogLevel level, bool color)
         fprintf(stdout, "%s ", formatted_timestamp);
     }
 
-    fprintf(stdout, "%s: %s\n", LogLevelToString(level), msg);
+    fprintf(stdout, "%8s: %s\n", LogLevelToString(level), msg);
```

Comment:
I had this when I was working on the logging branch actually, but I thought it looked funny since there is nearly always at least one space (when do you ever hit critical?), and removed it. But if you want it too, let's do it! :-)